sycl/test-e2e/MemorySanitizer/check_buffer_memset_memcpy.cpp

```diff
@@ -0,0 +1,64 @@
+// REQUIRES: linux, cpu || (gpu && level_zero)
```

Comment:
Just a note: we shouldn't change the test name after feature freeze, or we should remember to sync this change with jira config.